openssl/goopenssl.h

```diff
@@ -218,10 +218,16 @@ DEFINEFUNC(const GO_EVP_MD *, EVP_sha512, (void), ())
 DEFINEFUNC(const GO_EVP_MD *, EVP_md_null, (void), ())
 #if OPENSSL_VERSION_NUMBER < 0x30000000L
 DEFINEFUNCINTERNAL(int, EVP_MD_type, (const GO_EVP_MD *arg0), (arg0))
+DEFINEFUNCINTERNAL(size_t, EVP_MD_size, (const GO_EVP_MD *arg0), (arg0))
```

Comment:
`EVP_MD_size` and `EVP_MD_get_size` [docs](https://beta.openssl.org/docs/manmaster/man3/EVP_MD_get_size.html) say that these functions return an `int`, not a `size_t`.

Reply:
Good catch, fixed.